internal/ethapi/api.go

```diff
@@ -968,11 +968,11 @@ func (api *BlockChainAPI) GetBlockReceipts(ctx context.Context, blockNrOrHash rp
 // if stateDiff is set, all diff will be applied first and then execute the call
 // message.
 type OverrideAccount struct {
-	Nonce     *hexutil.Uint64              `json:"nonce"`
-	Code      *hexutil.Bytes               `json:"code"`
-	Balance   **hexutil.Big                `json:"balance"`
-	State     *map[common.Hash]common.Hash `json:"state"`
-	StateDiff *map[common.Hash]common.Hash `json:"stateDiff"`
```

Comment:
I think the reason was that we wanted to have everything optional, but missing or `null` json *can* decode into nil map, so should be fine. I think balance can also lose a pointer with the same logic.

Reply:
Thx sir. `Balance` field already remove a pointer now.